src/main/java/gov/cabinetoffice/gap/applybackend/service/GrantMandatoryQuestionService.java

```diff
@@ -161,22 +161,45 @@ public void addMandatoryQuestionsToSubmissionObject(final GrantMandatoryQuestion
     }
 
     public SubmissionSection buildOrganisationDetailsSubmissionSection(final GrantMandatoryQuestions mandatoryQuestions, final SubmissionSectionStatus sectionStatus) {
-        final SubmissionQuestion organisationName = mandatoryQuestionToSubmissionQuestion(MandatoryQuestionConstants.SUBMISSION_QUESTION_IDS.APPLICANT_ORG_NAME.toString(), mandatoryQuestions);
-        final SubmissionQuestion applicantType = mandatoryQuestionToSubmissionQuestion(MandatoryQuestionConstants.SUBMISSION_QUESTION_IDS.APPLICANT_TYPE.toString(), mandatoryQuestions);
-        final SubmissionQuestion organisationAddress = mandatoryQuestionToSubmissionQuestion(MandatoryQuestionConstants.SUBMISSION_QUESTION_IDS.APPLICANT_ORG_ADDRESS.toString(), mandatoryQuestions);
-        final SubmissionQuestion charityCommissionNumber = mandatoryQuestionToSubmissionQuestion(MandatoryQuestionConstants.SUBMISSION_QUESTION_IDS.APPLICANT_ORG_CHARITY_NUMBER.toString(), mandatoryQuestions);
-        final SubmissionQuestion companiesHouseNumber = mandatoryQuestionToSubmissionQuestion(MandatoryQuestionConstants.SUBMISSION_QUESTION_IDS.APPLICANT_ORG_COMPANIES_HOUSE.toString(), mandatoryQuestions);
+        final boolean isNonLimitedCompany = Objects.equals(mandatoryQuestions.getOrgType().toString(), "Non-limited company");
+        final boolean isIndividual = Objects.equals(mandatoryQuestions.getOrgType().toString(), "I am applying as an individual");
```

Comment:
might be worth defining these against the class since the second one is used in two places

Reply:
this function takes the mandatoryQuestions parameter which is used for identifying whether it's an individual - my java is rusty but I'm not sure it can be put against the class, can you give an example of how to do it?

Reply:
I can define the strings in constants for sure